sim/hunter/items.go

```diff
@@ -569,6 +570,124 @@ func init() {
 		hunter.ItemSwap.RegisterProc(SignetOfTheUnseenPath, procAura)
 	})
 
+	core.NewItemEffect(StringsOfFate, func(a core.Agent) {
+		hunter := a.(HunterAgent).GetHunter()
+
+		hasMeleeSpecialist := hunter.HasRune(proto.HunterRune_RuneBeltMeleeSpecialist)
+
+		// After using the bow active you cannot gain strands for 30sec.
+		noGainAura := hunter.GetOrRegisterAura(core.Aura{
+			Label:    "Strand Of Fate - No Stack Gain",
+			ActionID: core.ActionID{SpellID: 1231603},
+			Duration: time.Second * 30,
+		})
+
+		// Tracks the number of strands of fate, up to 5 stacks if you're ranged or 4 if you're using the melee version.
+		// TODO: Ranged Hunter's version lets them consume these stacks to move while shooting
+		stacksAura := hunter.GetOrRegisterAura(core.Aura{
```

Comment:
🔨 For all of these `GetOrRegisterAura`s, we should be able to just use `RegisterAura` instead since you can't have multiple of it equipped